docs/management/connectors/action-types/servicenow.asciidoc

```diff
@@ -23,13 +122,57 @@ creating a new {sn-itsm} connector or
 
 . Install
 https://store.servicenow.com/sn_appstore_store.do#!/store/application/7148dbc91bf1f450ced060a7234bcb88[Elastic for ITSM]
-from the {sn} Store.
+from the {sn} store.
+. <<servicenow-itsm-connector-privileges,Assign cross-scope privileges for the Elastic for ITSM app>>.
 . <<servicenow-itsm-connector-prerequisites-integration-user,Create a {sn} integration user and assign it the appropriate roles>>.
 . <<servicenow-itsm-connector-prerequisites-cors-rule,Create a Cross-Origin Resource Sharing (CORS) rule>>.
 . If you use open authorization (OAuth), you must also:
 .. <<servicenow-itsm-connector-prerequisites-rsa-key,Create an RSA keypair and add an X.509 Certificate>>.
 .. <<servicenow-itsm-connector-prerequisites-endpoint,Create an OAuth JWT API endpoint for external clients with a JWT Verifiers Map>>.
 
+[float]
+[[servicenow-itsm-connector-privileges]]
+==== Assign cross-scope privileges
+
+The Elastic for ITSM app requires specific cross-scope privilege records to run successfully:
+
+|===
+|Source scope|Target scope|Target name|Operation|Status
```

Comment:
I think we should also add the `Target Type` and maybe the `Application`. For the `Incident` the `Target Type` is different from the others.

<img width="1640" alt="Screenshot 2023-12-06 at 10 59 44 AM" src="https://github.com/elastic/kibana/assets/7871006/c69e7344-9c9e-4b2b-991f-73e36049d900">

Reply:
Thanks, I've edited the table in 500cb1f8b986ce07fcaf62b766a0cce0ad6b8ba1. Tables with this many columns don't work in our docs, so I've moved the source scope/application name and target scope into the introductory text.